apps/ejabberd/src/ejabberd.app.src

```diff
@@ -40,7 +40,8 @@
                   usec,
                   uuid,
                   xmerl,
-                  worker_pool
+                  worker_pool,
+                  hackney
```

Comment:
In many places we already use `fusco` for HTTP(s) queries to external services. What is the rational behind adding another HTTP lib?